pkg/api/vlabs/validate_test.go

```diff
@@ -934,13 +934,6 @@ func Test_Properties_ValidateNetworkPolicy(t *testing.T) {
 			"should error on cilium for windows clusters",
 		)
 	}
-
-	p.OrchestratorProfile.KubernetesConfig.NetworkPolicy = NetworkPolicyAntrea
-	if err := p.OrchestratorProfile.KubernetesConfig.validateNetworkPolicy(k8sVersion, true); err == nil {
```

Comment:
looking a few lines above it seems the k8sversion passed in is 1.7.9.  This should stay the same and check for 1.18+ should be added.

Reply:
Done.. Updated the patch

Reply:
@jsturtevant I explicitly added for 1.17. Do you still feel the need for 1.18 as 1.17 will cover validation failure